logstash-core/lib/logstash/util/worker_threads_default_printer.rb

```diff
@@ -6,8 +6,8 @@
 module LogStash module Util class WorkerThreadsDefaultPrinter
 
   def initialize(settings)
-    @setting = settings.fetch(:pipeline_workers, 0)
-    @default = settings.fetch(:default_pipeline_workers, 0)
+    @setting = settings.fetch('pipeline.workers', 0)
```

Comment:
.... from strings, to symbols, and back again! .... ( I actually don't care, but it is funny :) )